examples/react-native-vanilla/storybook/stories/index.js

```diff
@@ -13,13 +13,13 @@ storiesOf('Welcome', module).add('to Storybook', () => <Welcome showApp={linkTo(
 
 storiesOf('Button', module)
   .addDecorator(getStory => <CenterView>{getStory()}</CenterView>)
-  .add('with text', () => (
+  .add('with text', () =>
     <Button onPress={action('clicked-text')}>
       <Text>Hello Button</Text>
     </Button>
-  ))
-  .add('with some emoji', () => (
+  )
+  .add('with some emoji', () =>
```

Comment:
We can release, if we move some stuff over to 3.0.3:
https://github.com/storybooks/storybook/milestone/6